app/assets/javascripts/discourse/components/composer-editor.js.es6

```diff
@@ -253,7 +261,7 @@ export default Ember.Component.extend({
       const lastMatch = matchingPlaceholder[matchingPlaceholder.length - 1];
       const regex = new RegExp(regexString);
       const orderNr = regex.exec(lastMatch)[1]
```

Comment:
There's no point in executing the regex twice ;)

Reply:
Will address this in a separate commit since it's not related to this feature